src/js/events/CosmosPackagesActions.js

```diff
@@ -21,11 +21,12 @@ import {
 } from '../constants/ActionTypes';
 import AppDispatcher from './AppDispatcher';
 import Config from '../config/Config';
+import Util from '../utils/Util';
 
 const REQUEST_TIMEOUT = 10000;
 
-function getContentType(action, actionType) {
-  return `application/vnd.dcos.package.${action}-${actionType}+json;charset=utf-8;version=v1`;
+function getContentType(action, actionType, version = 'v1') {
```

Comment:
Yeah, maybe that is a good idea. I'll change that.
I guess I thought the two `v2`s were the odd ones out, but hopefully the rest will move to v2 soon too